mobile/lib/services/rest_api.dart

```diff
@@ -17,8 +17,8 @@ import 'package:uuid/uuid.dart';
 String addQueryParameters(Map<String, dynamic> queryParams, String url) {
   Map<String, dynamic> params = queryParams;
   params.remove("TOKEN");
-  String formattedUrl = '$url?TOKEN=${Config.airqoApiV2Token}';
-  params.forEach((key, value) => formattedUrl = "$formattedUrl&$key=$value");
+  String formattedUrl = '$url?';
+  params.forEach((key, value) => formattedUrl = "$formattedUrl$key=$value");
```

Comment:
I think we need the ampersand @BenjaminSsempala

Reply:
Thanks @NoahNsimbe